PayItForward.Data.Services/UsersService.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace PayItForward.Services.Data
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using PayItForward.Data;
+    using PayItForward.Services.Data.Abstraction;
+    using Dbmodel = PayItForward.Data.Models;
+
+    // TO DO :
+    // Bisness logic implementation later when Constructors are created
+
+    public class UsersService : IUsersService
+    {
+        private readonly UsersRepository<Dbmodel.User, string> usersRepo;
+
+        public UsersService()
+        {
```

Comment:
You should inject the repository into the service. https://www.dotnettricks.com/learn/dependencyinjection/implementation-of-dependency-injection-pattern-in-csharp